src/Components/CookieConsentManager/CookieConsentManager.tsx

```diff
@@ -0,0 +1,255 @@
+import { FC } from "react"
+import { ConsentManagerBuilder } from "@segment/consent-manager"
+import { getENV } from "Utils/getENV"
+import {
+  Box,
+  Button,
+  Checkbox,
+  Expandable,
+  Flex,
+  Join,
+  ModalDialog,
+  ReadMore,
+  Separator,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+import {
+  CATEGORIES,
+  DEFAULT_CATEGORIZATION,
+  DESTINATION_MAPPING,
+} from "Components/CookieConsentManager/categories"
+import { useMode } from "Utils/Hooks/useMode"
+import { CookieConsentBanner } from "Components/CookieConsentManager/CookieConsentBanner"
+
+const INITIAL_CUSTOM_PREFERENCES = CATEGORIES.reduce((prefs, category) => {
+  prefs[category.key] = category.key === "necessary"
+  return prefs
+}, {})
+
+export const CookieConsentManager: FC = () => {
+  const [mode, setMode] = useMode<
+    "Idle" | "Open" | "Allowing" | "Rejecting" | "Saving"
+  >("Idle")
+
+  return (
+    <ConsentManagerBuilder
+      shouldRequireConsent={() => true}
```

Comment:
Right now, for ease of testing, I'm forcing this, but this is where we'd use Segments [in-eu](https://github.com/segmentio/in-eu) package.